module/modmanager/manager.go

```diff
@@ -1010,10 +984,10 @@ func (m *module) checkReady(ctx context.Context, parentAddr string, logger loggi
 	req := &pb.ReadyRequest{ParentAddress: parentAddr}
 
 	// Wait for gathering to complete. Pass the entire SDP as an offer to the `ReadyRequest`.
-	if sdp, err := generateSDP(m.peerConn); err == nil {
-		req.WebrtcOffer = sdp
-	} else {
-		logger.Debugw("Unable to generate SDP for peerconn. Ignoring.", "err", err)
+	var err error
+	req.WebrtcOffer, err = m.sharedConn.GenerateEncodedOffer()
+	if err != nil {
+		logger.CWarnw(ctx, "Unable to generate offer for module PeerConnection. Ignoring.", "err", err)
```

Comment:
Are we allowed to add warn level logs for this optional feature?

Reply:
I'm not familiar of a style guide other than what I put into my logging scope. But IMO this is exactly what warn logs are for. An unexpected situation where the system retains functionality by sacrificing something else (e.g: performance).